lms/static/scripts/frontend_apps/components/LMSFilePicker.tsx

```diff
@@ -285,10 +291,10 @@ export default function LMSFilePicker({
     if (!file) {
       return;
     }
-    if (!file.type || file.type === 'File') {
-      onSelectFile(file);
-    } else {
+    if (file.type === 'Folder') {
       onChangePath(file);
+    } else {
+      onSelectFile(file);
```

Comment:
I flipped this condition so that resources with type `Page` work transparently for now, without having to explicitly reference pages here for now.